services/movie_session.py

```diff
@@ -0,0 +1,57 @@
+from db.models import MovieSession
+
+from services.movie import get_movie_by_id
+
+from services.cinema_hall import get_cinema_hall_by_id
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession.objects:
+    new_session = MovieSession.objects.create(

```

Comment:
You can safely put `return` right here, without introducing a variable.

Reply:
Thanks. Changed